convert2rhel/toolopts.py

```diff
@@ -205,37 +210,23 @@ def _register_options(self):
             " repositories by ID or glob. For more repositories to disable, use this option"
             " multiple times. This option defaults to all repositories ('*').",
         )
+        self._shared_options_parser.add_argument(
+            "-r",
+            "--restart",
+            help="Restart the system when it is successfully converted to RHEL to boot the new RHEL kernel."
+            " It has no effect when used with the 'analyze' subcommand.",
+            action="store_true",
+        )
```

Comment:
Also this one. The idea is that if we are doing an analysis, we don't need to restart the system after it finishes. 

Do you see any reason to make this different?

Reply:
As I was doing the changes, making the -r visible only for the convert subcommand and not the analyze subcommand was making it much more difficult. So I opted for making the code simpler and add the _"It has no effect when used with the 'analyze' subcommand"_ to the -r help message.